rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/clearingrequest/ClearingRequestController.java

```diff
@@ -165,6 +167,33 @@ public ResponseEntity<CollectionModel<EntityModel<ClearingRequest>>> getMyCleari
         return new ResponseEntity<>(resources, status);
     }
 
+    @Operation(
+            summary = "Add a new comment to a clearing request.",
+            description = "Create a new comment for the clearing request.",
+            tags = {"ClearingRequest"}
+    )
+    @PreAuthorize("hasAuthority('WRITE')")
+    @RequestMapping(value = CLEARING_REQUEST_URL + "/{id}/comments", method = RequestMethod.POST)
+    public ResponseEntity<?> addComment(
+            @Parameter(description = "ID of the clearing request")
+            @PathVariable("id") String crId,
+            @RequestBody Comment comment,
```

Comment:
Please add OpenAPI documentation for the comment parameter using `@Parameter` annotation as well.